src/components/IOUConfirmationList.js

```diff
@@ -0,0 +1,248 @@
+    })).isRequired,
+
+    /* Onyx Props */
+
+    // The personal details of the person who is logged in
+    myPersonalDetails: PropTypes.shape({
+
+        // Display name of the current user from their personal details
+        displayName: PropTypes.string,
+
+        // Avatar URL of the current user from their personal details
+        avatar: PropTypes.string,
+
+        // Primary login of the user
+        login: PropTypes.string,
+    }).isRequired,
+
+    // Holds data related to IOU view state, rather than the underlying IOU data.
+    iou: PropTypes.shape({
+
+        // Whether or not the IOU step is loading (creating the IOU Report)
+        loading: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    iou: {},
+    onUpdateComment: null,
+    comment: '',
+};
+
+class IOUConfirmationList extends Component {
+    /**
+     * Returns the sections needed for the OptionsSelector
+     *
+     * @param {Boolean} maxParticipantsReached
+     * @returns {Array}
+     */
+    getSections() {
+        const sections = [];
+
+        if (this.props.hasMultipleParticipants) {
+            const formattedMyPersonalDetails = getIOUConfirmationOptionsFromMyPersonalDetail(
+                this.props.myPersonalDetails,
+
+                // convert from cent to bigger form
+                `$${this.calculateAmount(true) / 100}`,
+            );
+            const formattedParticipants = getIOUConfirmationOptionsFromParticipants(this.props.participants,
+                `$${this.calculateAmount() / 100}`);
+
+            sections.push({
+                title: 'WHO PAID?',
+                data: formattedMyPersonalDetails,
+                shouldShow: true,
+                indexOffset: 0,
+            });
+            sections.push({
+                title: 'WHO WAS THERE?',
+                data: formattedParticipants,
+                shouldShow: true,
+                indexOffset: 0,
+            });
+        } else {
+        // $ should be replaced by currency symbol once available
+            const formattedParticipants = getIOUConfirmationOptionsFromParticipants(this.props.participants,
+                `$${this.props.iouAmount}`);
+
+            sections.push({
+                title: 'TO',
+                data: formattedParticipants,
+                shouldShow: true,
+                indexOffset: 0,
+            });
+        }
+        return sections;
+    }
+
+    /**
+     * Gets splits for the transaction
+     *
+     * @returns {Array}
+     */
+    getSplits() {
+        const splits = this.props.participants.map(participant => ({
+            email: participant.login,
+
+            // we should send in cents to API
+            amount: this.calculateAmount(),
+        }));
+
+        splits.push({
+            email: this.props.myPersonalDetails.login,
+
+            // the user is default and we should send in cents to API
+            amount: this.calculateAmount(true),
```

Comment:
Can you leave a comment here to say that USD is temporary and that we should support all currencies with later changes?